packages/create-tina-app/src/index.ts

```diff
@@ -20,10 +20,12 @@ import { PackageManager, PKG_MANAGERS } from './util/packageManagers';
 import validate from 'validate-npm-package-name';
 import * as ascii from './util/asciiArt';
 import { THEMES } from './themes';
+import { useState } from 'react';
```

Comment:
I don't think we need react for this cli, seems overkill for tracking 1 piece of state